src/ir.h

```diff
@@ -66,13 +66,17 @@ struct Const {
   struct F64 {};
 
   // Keep the default constructor trivial so it can be used as a union member.
-  Const() = default;
+  Const() {
+    this->type = Type::Any;
+    this->u32 = 0;
+    this->u64 = 0;
+    this->f32_bits = 0;
+    this->f64_bits = 0;
+  }
   Const(I32, uint32_t);
   Const(I64, uint64_t);
   Const(F32, uint32_t);
   Const(F64, uint64_t);
-  // TODO(binji): Make these default arguments on the constructors above when
```

Comment:
If you remove, can you remove the constructors and make the above into default args?